titus-server-runtime/src/main/java/com/netflix/titus/runtime/connector/common/replicator/AbstractReplicatorEventStream.java

```diff
@@ -54,7 +54,11 @@ protected AbstractReplicatorEventStream(DataReplicatorMetrics metrics, TitusRunt
                 })
                 .doOnCancel(metrics::disconnected)
                 .doOnError(error -> {
-                    logger.warn("[{}] Connection to the event stream terminated with an error: {}", getClass().getSimpleName(), error.getMessage(), error);
+                    if (logger.isDebugEnabled()) {
+                        logger.debug("[{}] Connection to the event stream terminated with an error: {}", getClass().getSimpleName(), error.getMessage(), error);
+                    } else {
+                        logger.warn("[{}] Connection to the event stream terminated with an error: {}", getClass().getSimpleName(), error.getMessage());
```

Comment:
I prefer to always log at `warn` level, and have an additional line for debug. Otherwise `grep WARN ... ` would miss this log line if debug level is set.

Reply:
sure, will change